taskcluster/ci/build/kind.yml

```diff
@@ -0,0 +1,97 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - focus_android_taskgraph.transforms.build:transforms
+    - taskgraph.transforms.job:transforms
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - toolchain
+
+job-defaults:
+    apk-artifact-template:
+        type: file
+        name: '{fileName}.apk'
+        path: '/opt/focus-android/app/build/outputs/apk/{gradle_build_name}/{gradle_build_type}'
+        # path: '/builds/worker/checkouts/src/app/build/outputs/apk/{gradle_build_type}'
+    fetches:
+        toolchain:
+            - android-sdk-linux
+            - android-gradle-dependencies
+    run:
+        using: gradlew
+        use-caches: false
+        workdir: '/builds/worker'
+    run-on-tasks-for: []
+    treeherder:
+        kind: build
+        symbol: B
+        platform: android-all/opt
+        tier: 1
+    worker-type: b-android
+    worker:
+        docker-image: {in-tree: base}
+        max-run-time: 7200
+        chain-of-trust: true
+
+jobs:
+    focus-debug:
+        description: 'Focus debug build from source code'
+        run-on-tasks-for: [github-pull-request, github-push]
+        run:
+            gradle-build-type: debug
+            gradle-build-name: focusDebug
+        treeherder:
+            symbol: debug(Bf)
+
+    klar-debug:
+        description: 'Klar debug build from source code'
+        run-on-tasks-for: [github-pull-request, github-push]
+        run:
+            gradle-build-type: debug
+            gradle-build-name: klarDebug
+        treeherder:
+            symbol: debug(Bkl)
+
+    focus-release:
+        description: 'Release Focus build for debugging'
+        run-on-tasks-for: [github-pull-request]
```

Comment:
I'm pretty sure we'll want to remove this and the other release/beta/nightly builds from pull requests before this lands. (I'm guessing this was just for testing purposes - I just want to make sure we don't land it by accident.)

Reply:
It was intentional. @pocmo said he wanted to run all builds so that they're not caught by a surprise if something breaks in one of these builds but not on debug. But I believe he said 'on merge' was also okay (please correct me if I'm wrong Sebastian).